config/packages/ezplatform_welcome_page.yaml

```diff
@@ -0,0 +1,12 @@
+# You can safely delete this file to disable welcome page
+ezplatform:
+    system:
+        site:
+            content_view:
+                full:
+                    welcome_page:
+                        template: "@ezdesign/full/welcome_page.html.twig"
+                        match:
+                            Id\Location: 2
+                        params:
+                            project_dir: '%kernel.project_dir%'
```

Comment:
You need to merge ezsystems/ezplatform#515 into `ezplatform-ee:master` first, then rebase so this file disappears from the diff as it's the same as for OSS. Otherwise you'll get a conflict on squash, or the next person merging horizontally something else will get it.